edge/pkg/edged/edged.go

```diff
@@ -64,13 +68,13 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/util/format"
 	"k8s.io/kubernetes/pkg/kubelet/util/queue"
 	"k8s.io/kubernetes/pkg/kubelet/volumemanager"
-	"k8s.io/kubernetes/pkg/scheduler/schedulercache"
-	kubeio "k8s.io/kubernetes/pkg/util/io"
+	schedulercache "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 	"k8s.io/kubernetes/pkg/util/mount"
 	"k8s.io/kubernetes/pkg/volume"
 	"k8s.io/kubernetes/pkg/volume/configmap"
-	"k8s.io/kubernetes/pkg/volume/empty_dir"
-	"k8s.io/kubernetes/pkg/volume/host_path"
+	"k8s.io/kubernetes/pkg/volume/csi"
+	emptydir "k8s.io/kubernetes/pkg/volume/emptydir"
```

Comment:
do we need the package identifier , since by default the last name in the package structure will be used.

Reply:
oh..... Thanks. will remove:)